src/test/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackServiceTest.kt

```diff
@@ -0,0 +1,112 @@
+package com.hibob.academy.employeeFeedback.service
+
+import com.hibob.academy.employeeFeedback.dao.*
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import java.sql.Date
+
+class FeedbackServiceTest{
+
+    private val feedbackDao = mock<FeedbackDao>{}
+    private val employeeDao = mock<EmployeeDao>{}
+    private val feedbackService = FeedbackService(feedbackDao, employeeDao)
+
+    @Test
+    fun `Submit identified feedback successfully`(){
+        val anonymity = AnonymityType.IDENTIFIED
+        val feedback = "I'm very happy with my workspace, i'm treated well"
+        val feedbackForSubmission = FeedbackForSubmission(12,1, anonymity, feedback)
+        val output = feedbackService.submitFeedback(12, 1, anonymity, feedback)
+        assertTrue(output)
+        verify(feedbackDao).submitFeedback(feedbackForSubmission)
+    }
+
+    @Test
+    fun `Submit anonymous feedback successfully`(){
+        val anonymity = AnonymityType.ANONYMOUS
+        val feedback = "I'm very happy with my workspace, i'm treated well"
+        val feedbackForSubmission = FeedbackForSubmission(null, 1, anonymity, feedback)
+        val output = feedbackService.submitFeedback(12, 1, anonymity, feedback)
```

Comment:
unless you are testing that this feedback was saved as anonymous there's no difference between this test and the one before.

Add tests that check the logic that is done in the service itself. If the service doesn't do different logic here, no need for this test